### PR TITLE
New Koa configuration

### PR DESCRIPTION
**Pull request by arthurM2x**
Includes domain creation for better breadcrumbs handling and creation of transaction per request

### CODE REVIEW

File: src/collections/_documentation/performance-monitoring/configuration/koa.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+###Koa
```
Comment (dashed) [inline src/collections/_documentation/performance-monitoring/configuration/koa.md:1]:
```suggestion
### Koa
```

File: src/collections/_documentation/performance-monitoring/configuration/koa.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+Creates and attach a transaction to each context
```
Comment (dashed) [inline src/collections/_documentation/performance-monitoring/configuration/koa.md:2]:
```suggestion
### Koa

Creates and attach a transaction to each context
```

File: src/collections/_documentation/performance-monitoring/configuration/koa.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      description: route,
+      op: 'myMiddleware',
+    })
+  }
+  await myExpensiveOperation()
+  if (span) {
+    span.finish()
+  }
+  return next()
+}
```
Comment (dashed) [inline src/collections/_documentation/performance-monitoring/configuration/koa.md:100]:
````suggestion
}
```
````

File: src/collections/_documentation/performance-monitoring/configuration/koa.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      return Sentry.Handlers.parseRequest(event, ctx.request)
+    })
+    Sentry.captureException(err)
+  })
+})
+// the rest of your app
+```
+
+#### Subsequent manual child transactions
+The following example creates a transaction for a part of the code that contains an expensive operation, and sends the result to Sentry, you will need to use the transaction stored in the context
```
Comment (dashed) [inline src/collections/_documentation/performance-monitoring/configuration/koa.md:83]:
```suggestion
#### Subsequent manual child transactions

The following example creates a transaction for a part of the code that contains an expensive operation, and sends the result to Sentry, you will need to use the transaction stored in the context
```

**Comment by dashed** [comment]
@arthurM2x You may need to update the jest snapshots by running `yarn test -u`. 

Thanks for adding this example 👍  I was able to run the example locally.

**Comment by arthurM2x** [comment]
@dashed looks like I don't need to update snapshots?

**Comment by dashed** [comment]
@arthurM2x you're right. Looks like the travis checks weren't happy about the whitespace issues.

**Pull request merged**